pkg/storage/cacher.go

```diff
@@ -467,8 +467,8 @@ type cacheWatcher struct {
 
 func newCacheWatcher(resourceVersion uint64, initEvents []watchCacheEvent, filter FilterFunc, forget func(bool)) *cacheWatcher {
 	watcher := &cacheWatcher{
-		input:   make(chan watchCacheEvent, 10),
-		result:  make(chan watch.Event, 10),
+		input:   make(chan watchCacheEvent, 100),
```

Comment:
This explanation sounds reasonable to me. 

Then, in k8s, the default `NodeStatusUpdateFrequency` is 10s, which mean kubelet will update it's status every 10s. In k8s 1.2, we have 1000 nodes. In the worst case, all the 1000 nodes update their status simultaneous. The the throughput would be 1000/second. 10 is too small if a client was watching /registry/nodes/